### PR TITLE
GEN-658: Update the ingestion schedular with new design 

### LINKED ISSUE #17209: Implement the pipeline scheduler new mocks
Implement the new ingestion scheduler mocks [here](https://www.figma.com/proto/KMwnWm5vnfPNZiru4z3lbi/OpenMetadata-Final-Screens?page-id=&node-id=4290-46098&viewport=-195%2C-2315%2C0.16&t=9vYk5W7mZphIEQ4e-1&scaling=scale-down-width&content-scaling=fixed&starting-point-node-id=4377%3A89873)

### LINKED ISSUE #18019: Ingestion pipeline search should only return relevant results.
**Affected module**
UI

**Describe the bug**
When searching for ingestion pipelines on the service details page, the results shown, contain ingestion pipelines from other services as well.

https://github.com/user-attachments/assets/cd4d2fb2-d351-4226-9660-5d66682e78ef

**Expected behavior**
The search should only display the pipeline from the respective service.

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### LINKED ISSUE #18188: bug: Ingestion schedule form not properly filled when editing existing ingestion
**Affected module**
UI

**Describe the bug**
When I add ingestion process with some schedule (say every Day at 5 AM) I click Submit and the change is persisted. I can see in Airflow

**To Reproduce**

- Register ingestion with Schedule: every day 5 pm
- Submit change
- Go to Edit Ingestion form (schedule tab)
- The form contains Schedule: Every Week, Monday midnight

**Expected behavior**
- The form contains Schedule I have defined previously: every day 5 pm

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8] 1.5.4
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`] 1.5.4.0

**Additional context**

### PR DESCRIPTION
**Pull request by aniketkatkar97**
Fixes #17209 
Fixes #18188 
Fixes #18019

I worked on the following:

https://github.com/user-attachments/assets/ad735265-d021-46ee-a732-51d00ff7ec2c

- [x] Update the scheduler according to the new mocks
- [x] Update the new scheduler in all the places
- [x] Make the `Daily` the default schedule except for the Data Insights Report application which will have `Week` as the default schedule
- [x] Add a validation check for the custom cron expression to prevent cron from setting intervals that are more frequent than hourly.
- [x] Fix the issue where the set schedule was not reflecting on the edit ingestion page.

- [x] Fix the ingestion pipeline search returning pipelines from other services as well.

**Before:**

https://github.com/user-attachments/assets/fdd5c628-2c8d-48dc-84b0-731026753a80

**After:**

https://github.com/user-attachments/assets/69d745e4-7888-49f6-b77a-ac83c6e26f37

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/DataQuality/AddDataQualityTest/TestSuiteIngestion.tsx [hunk 1]
```diff
@@ -211,9 +203,7 @@ const TestSuiteIngestion: React.FC<TestSuiteIngestionProps> = ({
       displayName: data.name ?? ingestionPipeline.displayName,
       airflowConfig: {
         ...ingestionPipeline?.airflowConfig,
-        scheduleInterval: isEmpty(data.repeatFrequency)
-          ? undefined
-          : data.repeatFrequency,
+        scheduleInterval: isEmpty(data.cron) ? undefined : data.cron,
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/DataQuality/AddDataQualityTest/TestSuiteIngestion.tsx:206]: In which case we have an empty cron? can we avoid that so we don't need such a check?

File: openmetadata-ui/src/main/resources/ui/src/components/Settings/Services/AddIngestion/AddIngestion.component.tsx [hunk 2]
```diff
@@ -168,9 +162,7 @@ const AddIngestion = ({
 
     const ingestionDetails: CreateIngestionPipeline = {
       airflowConfig: {
-        scheduleInterval: isEmpty(scheduleInterval)
-          ? undefined
-          : scheduleInterval,
+        scheduleInterval: isEmpty(extraData.cron) ? undefined : extraData.cron,
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Settings/Services/AddIngestion/AddIngestion.component.tsx:165]: same

File: openmetadata-ui/src/main/resources/ui/src/constants/Ingestions.constant.ts [hunk 3]
```diff
@@ -53,3 +53,8 @@ export const PIPELINE_TYPE_LOCALIZATION = {
 export const DBT_CLASSIFICATION_DEFAULT_VALUE = 'dbtTags';
 
 export const DEFAULT_PARSING_TIMEOUT_LIMIT = 300;
+
+export const DEFAULT_SCHEDULE_CRON = '0 0 * * *';
+export const DEFAULT_CRON_MIN_VALUE = 0;
+export const DEFAULT_CRON_HOUR_VALUE = 0;
+export const DEFAULT_CRON_WEEK_VALUE = 1;
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/constants/Ingestions.constant.ts:60]: Isn't this should be Schedular constant?

File: openmetadata-ui/src/main/resources/ui/src/enums/Schedular.enum.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+export enum SchedularOptions {
+  SCHEDULE = 'schedule',
+  ON_DEMAND = 'on-demand',
+}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/enums/Schedular.enum.ts:17]: Can we club cron and schedular enum?

Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/enums/Schedular.enum.ts:17]: Sure

File: openmetadata-ui/src/main/resources/ui/src/components/DataQuality/AddDataQualityTest/TestSuiteIngestion.tsx (hunk 1 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/DataQuality/AddDataQualityTest/TestSuiteIngestion.tsx:206]: My bad, we have a check for that input not to be empty. I'll remove the check.

Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/components/DataQuality/AddDataQualityTest/TestSuiteIngestion.tsx:206]: It seems you miss to push the changes here?

Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/DataQuality/AddDataQualityTest/TestSuiteIngestion.tsx]: pushed the changes

**Review by chirag-madlani — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**